sdk/src/process_instruction.rs

```diff
@@ -309,6 +313,19 @@ pub mod stable_log {
         ic_logger_msg!(logger, "Program log: {}", message);
     }
 
+    /// Log return data as from the program itself.
```

Comment:
Just to be complete please also document here that the `Program return data:` will not be present if no return data was provided (as opposed to an empty `<program-generated-in-base64>` value)

Reply:
So with the program_id, the return_data becomes an Option<>. Now it is present, even if it is empty.